benchalerts/benchalerts/pipeline_steps/conbench.py

```diff
@@ -11,17 +12,46 @@
 ConbenchClient = LegacyConbenchClient
 
 
-class GetConbenchZComparisonStep(AlertPipelineStep):
-    """An ``AlertPipeline`` step to get information from Conbench comparing the runs on
-    a contender commit to their baselines, using a z-score threshold. This is always the
-    first step of the pipeline.
+CONBENCH_ENV_VAR_HELP = """
+
+    Notes
+    -----
+    Environment variables
+    ~~~~~~~~~~~~~~~~~~~~~
+    ``CONBENCH_URL``
+        The URL of the Conbench server. Only required if ``conbench_client`` is not
+        provided.
+    ``CONBENCH_EMAIL``
+        The email to use for Conbench login. Only required if ``conbench_client`` is not
+        provided and the server is private.
+    ``CONBENCH_PASSWORD``
+        The password to use for Conbench login. Only required if ``conbench_client`` is
+        not provided and the server is private.
+"""
+
+
+class BaselineRunCandidates(Enum):
+    """Types of baselines available from `/api/runs/{run_id}` in the
+    `candidate_baseline_runs` field.
+    """
+
+    fork_point = "fork_point"
+    head_of_default = "head_of_default"
+    parent = "parent"
+
+
+class GetConbenchZComparisonForRunsStep(AlertPipelineStep):
+    (
```

Comment:
Unfortunately this doesn't count as a docstring, so the docs don't show up on hover in IDEs or in the rendered HTML on our GitHub Page.
```py
>>> TEST = "string"
>>> class a:
...     """docstring"""
...     pass
...
>>> class b:
...     ("""doc""" + TEST)
...     pass
...
>>> a.__doc__
# 'docstring'
>>> b.__doc__ is None
# True
```
f-strings don't work either, only postprocessing after the class definition. You could do something like
```py
GetConbenchZComparisonForRunsStep.__doc__ += CONBENCH_ENV_VAR_HELP
```
at the end of the file if you'd like.

Reply:
This is why I didn't make the docstrings DRY earlier. We could make a separate issue for a decorator like other libraries tend to do.

Reply:
Oh this is annoying and against my expectations; I'll revert. Putting it at the end of the file would make it too inobvious it's there when reading the code imo.